<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalomeOptimizationDialog</class>
 <widget class="QDialog" name="SalomeOptimizationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configuration of the variables to be optimized</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="noIterationsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>228</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Number of optimization iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="noIterationsLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>113</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectTemplateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>228</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Select Template</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="templateComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="variablesListTable">
     <column>
      <property name="text">
       <string>Changed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variable Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actual Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Optimize</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleSelected">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configure the values for the selected variables in each iteration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="selectedVarListTable">
     <column>
      <property name="text">
       <string>Variable Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actual Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string>Create Mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="consoleTextEdit"/>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="finishButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
